backend/controllers/listingsController.js

```diff
@@ -0,0 +1,37 @@
+const Listing = require('../models/listingModel')
+const mongoose = require('mongoose')
+
+
+const getAllListings = async (req, res) => {
+  const listings = await Listing.find({}).sort({createdAt: -1});
+  res.status(200).json(listings);
+}
+
+const createListing = async (req, res) => {
+  const {organisationName, title, description, address, neededByDate} = req.body;
+  let emptyFields = [];
+  if (!organisationName){
```

Comment:
Is an if/else statement necessary? Could this be done with a map?

Reply:
I've refactored it to a map and have pushed to the branch. Can see changes on the branch.